neural_speed/convert/convert_mistral.py

```diff
@@ -179,6 +180,8 @@ def loadHFTransformerJson(model: 'LazyModel', config_path: Path) -> 'Params':
         ffn_hidden_size = config["intermediate_size"]
         rms_norm_eps = config["rms_norm_eps"]
         rope_theta = config["rope_theta"] if "rope_theta" in config else 10000
+        rope_scale = config["factor"] if "factor" in config else 1
```

Comment:
mistral should align to llama

Reply:
please help update convert convert_quantized_llama.py and convert_quantized_mistral.py